### ISSUE TITLE
List Shortcuts is on `Ctrl+F1` instead of `?`

### ISSUE THREAD

**Issue by KarlZeilhofer** [KiCad/kicad-doc#758]
I found this to be true on KiCad 5.1.4 on Linux Mint 19.2

**Comment by ciampix** [comment]
Yes confirmed... we should probably standardize on "?" or Ctrl+F1 or leave F1 for help on all apps of the KiCad suite...
